docs/community/out-there.rst

```diff
@@ -4,7 +4,7 @@ Integrations
 Python for iOS
 --------------
 
-Requests is built into the wonderful `Python for iOS <https://itunes.apple.com/us/app/python-2.7-for-ios/id485729872?mt=Python8>`_ runtime!
+Requests is built into the wonderful `Python for iOS <https://apps.apple.com/us/app/python-2-5-for-ios/id577916777>`_ runtime!
```

Comment:
I believe this is a similar app by the same developer, but it's python 2.5 (long deprecated), and appears to be only old version of iOS so won't work on recent devices anyway.

I was tempted to just delete the whole 'Python for iOS' section here, but I thought I would see what a maintainer thought before deleting the whole section.